packages/manager/apps/sign-up/src/form/form.controller.js

```diff
@@ -96,6 +96,7 @@ export default class SignUpFormAppCtrl {
           this.saveError = error;
         });
     }
+    if (this.needkyc) this.goToKycDocumentUploadPage();
```

Comment:
**nitpick:** This can be moved above line 90. So that you can avoid the check `!this.needKyc` in line 90

Reply:
Done Anoop